src/ape/contracts/base.py

```diff
@@ -193,15 +206,40 @@ def __repr__(self) -> str:
 
     @property
     def address(self) -> AddressType:
+        """
+        The address of the contract.
+
+        Returns:
+            :class:`~ape.types.AddressType`
+        """
         return self._address
 
     def __dir__(self) -> List[str]:
-        # This displays methods to IPython on `c.[TAB]` tab completion
+        """
+        Display methods to IPython on ``c.[TAB]`` tab completion.
+
+        Returns:
+            list[str]
+        """
         return list(super(AddressAPI, self).__dir__()) + [
             abi.name for abi in self._contract_type.abi
         ]
 
     def __getattr__(self, attr_name: str) -> Any:
+        """
+        Access a method or property on the contract using ``.`` access.
+
+        Usage example::
+
+            result = contract.Vote()  # Implies a method named Vote exists on the contract.
```

Comment:
Methods are usually "camelCase"
```suggestion
            result = contract.vote()  # Implies a method named "vote" exists on the contract.
```